Mage.Sets/src/mage/cards/h/HenzieToolboxTorre.java

```diff
@@ -0,0 +1,191 @@
+package mage.cards.h;
+
+import mage.MageInt;
+import mage.MageObject;
+import mage.abilities.Ability;
+import mage.abilities.common.SimpleStaticAbility;
+import mage.abilities.effects.AsThoughEffect;
+import mage.abilities.effects.ContinuousEffectImpl;
+import mage.abilities.effects.common.cost.CostModificationEffectImpl;
+import mage.abilities.keyword.BlitzAbility;
+import mage.cards.Card;
+import mage.cards.CardImpl;
+import mage.cards.CardSetInfo;
+import mage.constants.*;
+import mage.filter.FilterObject;
+import mage.filter.predicate.mageobject.ManaValuePredicate;
+import mage.game.Game;
+import mage.game.permanent.Permanent;
+import mage.game.stack.Spell;
+import mage.game.stack.StackObject;
+import mage.players.Player;
+import mage.util.CardUtil;
+import mage.watchers.common.CommanderPlaysCountWatcher;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+
+/**
+ * @author Alex-Vasile
+ */
+public class HenzieToolboxTorre extends CardImpl {
+
+    public HenzieToolboxTorre(UUID ownerId, CardSetInfo setInfo) {
+        super(ownerId, setInfo, new CardType[]{CardType.CREATURE}, "{B}{R}{G}");
+
+        addSuperType(SuperType.LEGENDARY);
+        addSubType(SubType.DEVIL, SubType.ROGUE);
+
+        this.power = new MageInt(3);
+        this.toughness = new MageInt(3);
+
+        // Each creature spell you cast with mana value 4 or greater has blitz.
+        // The blitz cost is equal to its mana cost.
+        //      (You may choose to cast that spell for its blitz cost.
+        //       If you do, it gains haste and “When this creature dies, draw a card.”
+        //       Sacrifice it at the beginning of the next end step.)
+        this.addAbility(new SimpleStaticAbility(new HenzieToolboxTorreGainBlitzEffect()));
+
+        // Blitz costs you pay cost {1} less for each time you’ve cast your commander from the command zone this game.
+        this.addAbility(new SimpleStaticAbility(new HenzieToolboxTorreBlitzDiscountEffect()));
+    }
+
+    private HenzieToolboxTorre(final HenzieToolboxTorre card) {
+        super(card);
+    }
+
+    @Override
+    public HenzieToolboxTorre copy() {
+        return new HenzieToolboxTorre(this);
+    }
+}
+
+// Must use custom effect rather than GainAbilitySpellsEffect since the BlitzAbility requires the card object passed to the initializer
+class HenzieToolboxTorreGainBlitzEffect extends ContinuousEffectImpl {
```

Comment:
This will allow the card to be cast with blitz, but it will not gain haste or the sacrifice triggered ability when its cast. Any thoughts? @theelk801

Reply:
sorry I forgot to respond to this, I need to take a closer look at the code